### ISSUE TITLE
XDA feeder only retrieves the first post of the last 3 pages

### ISSUE THREAD

**Issue by klo2k** [klo2k/php-rss#3]
XDA seems to have inserted some new JS code after the first post, which gets incorrectly parsed as HTML by loadHTML(), messing up the XPath query to get $postDivs - it only returns 1 node as a result.

**Comment by klo2k** [comment]
e.g:

<div id="posts">
...
[first post]
...
            <div class="post-content-container">
                <div class="postbit-userinfo-cell">
                </div>

```
            <div class="postbit-content">
                <!-- AD LOCATION: showthread_afterfirstpost_desktop --><script type='text/javascript'>
if(!is_mobile)
{
    if (xda_ad_choice == "purch")
    {
        document.write('<div id="purch_Y_L_0_1" class="purchad" style="text-align:left; min-height:90px; max-height:250px; overflow:hidden;"></div>');
    }
    else if (xda_ad_choice == "sortable")
    {
        document.write('\x3Cscript src="//tags-cdn.deployads.com/a/xda-developers.com.js" async >\x3C/script>');
        document.write('<div class="ad-tag" data-ad-name="forum_incontent_1" data-ad-size="728x90" style="text-align:left; min-height:90px; max-height:250px; overflow:hidden;"></div>');

        (deployads = window.deployads || []).push({});
    }
}
```

</script>



```
            </div>
        </div>
```

**Comment by klo2k** [comment]
Testing fix....
[xda.php.txt](https://github.com/klo2k/php-rss/files/183729/xda.php.txt)

**Comment by klo2k** [comment]
Fixed in 18627e3ab15765e574cd8f545be1011a111f6d6a
